x-pack/plugins/observability_solution/inventory/public/hooks/use_detail_view_redirect.ts

```diff
@@ -65,9 +65,6 @@ export const useDetailViewRedirect = () => {
       if (isBuiltinEntityOfType('service', entity)) {
         return serviceOverviewLocator?.getRedirectUrl({
           serviceName: identityFieldsValue[identityFields[0]],
-          environment: entity.service?.environment
-            ? castArray(entity.service?.environment)[0]
-            : undefined,
         });
```

Comment:
This change was agreed with product to avoid inconsistent data when accessing the service detail view with an environment other than "All" by default.